src/test/resources/features/MenuSections.feature

```diff
@@ -0,0 +1,12 @@
+Feature: As a user I want to get ability to use menu sections
+
+  Scenario Outline: Tools sections should contain appropriate columns
+    Given the user opens Cucumber website
```

Comment:
This line can be used as Background

Reply:
Refactored, thank you for pointing.
Please see in the following commit.